recipe/build-python.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 set -ex
 
-# do not use the same folder as global build.sh
+# do not use the same folder as global build.sh,
+# but delete it from any previous python builds
+rm -rf build-py || true
 mkdir build-py
```

Comment:
you could remove both lines
```sh
mkdir build-py
cd build-py
```
and the one below `cd ..`
and use:
```sh
cmake -G .... -S . -B build-py
cmake --build build-py ...
```

Reply:
True, but that requires implicit knowledge about CMake does things, and I prefer self-documenting code that uses as little "magic" as possible.